r/rstudio/Dockerfiles/v1.3.1073/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM     ppc64le/adoptopenjdk:8-openj9-bionic
```

Comment:
can we use the multiarch manifest here? that is adoptopenjdk:8-openj9-bionic instead of ppc64le/adoptopenjdk:8-openj9-bionic

Reply:
I can test with it.

Reply:
yes, and the reason for that is most likely that that dockerfile was written before multiarch manifest support for official docker images was introduced - if you see the more recent Dockerfiles in this repo, you should find some good examples.